.gitignore

```diff
@@ -4,3 +4,5 @@ lib
 es6
 docs/_site/
 packages/ra-example/static
+# IDE 
+.idea
```

Comment:
Please keep those developer specific ignore to your global gitignore: https://help.github.com/articles/ignoring-files/#create-a-global-gitignore

Reply:
Nice one! Didn't know of the existence of this option.